packages/pipeline/src/utils/transformers/number_to_bigint.ts

```diff
@@ -9,8 +9,12 @@ const decimalRadix = 10;
 // https://github.com/typeorm/typeorm/issues/2400 for more information.
 export class NumberToBigIntTransformer implements ValueTransformer {
     // tslint:disable-next-line:prefer-function-over-method
-    public to(value: number): string {
-        return value.toString();
+    public to(value: number): string | null {
+        if (typeof value !== 'number') {
```

Comment:
If `value` is not a `number` something is spectacularly wrong. IMO we should throw an error instead of just returning `null`.

If `value` is sometimes `null` or `undefined` we should add a more specific check for that.